pkg/operator/ceph/object/realm/controller.go

```diff
@@ -198,6 +237,52 @@ func (r *ReconcileObjectRealm) createCephRealm(realm *cephv1.CephObjectRealm) (r
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileObjectRealm) createRealmKeys(realm *cephv1.CephObjectRealm) (reconcile.Result, error) {
+	accessKey, err := mgr.GeneratePassword(accessKeyLength)
+	if err != nil {
+		return reconcile.Result{}, errors.Wrapf(err, "access key failed to generate")
+	}
+	accessKey = base64.StdEncoding.EncodeToString([]byte(accessKey))
```

Comment:
Please add a comment about why we're generating the access and secret keys with the two-step process